shiny/ui/_html_deps_external.py

```diff
@@ -47,7 +49,19 @@ def bootstrap_theme_deps(theme: str | Path | ThemeProvider | None) -> TagList:
             + f"but received `theme` with type {type(theme)}."
         )
 
-    return TagList(deps_bootstrap, deps_theme)
+    return TagList(
+        deps_bootstrap,
+        deps_theme,
+        # These next dependencies have their CSS bundled in the main Bootstrap CSS
+        # but are also included at the component level. This page-level dependency will
+        # win in Shiny apps, but the component-level dependencies will win in other
+        # contexts where we don't have a page function.
```

Comment:
Might also be good to add a bit of more context here that we're thinking of this as mainly a workaround for "legacy Sass" in 3rd party libraries that want Bootstrap Sass variables "baked-in" to their CSS. And, in the future, we can and should try out best to avoid this by leveraging CSS variables for theming